File: Frontend/src/app/contracts/room.ts
import { Idea } from './idea';
import { Owner } from './owner';
import { User } from './user';

export class Room {
    id: number;
    name: string;
    timemarks: Date[];
    description: string;
    owner: Owner;
    ideas: Idea[];
    users: User[];
  }

File: Backend/PlanYourParty.WebApi.Contracts/PlanYourParty.WebApi.Contracts/Room.cs
using System;
using System.Collections.Generic;

namespace PlanYourParty.WebApi.Contracts
{
    public class Room
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public IEnumerable<DateTime> Timemarks { get; set; }
        public Owner Owner { get; set; }
        public IEnumerable<User> Users { get; set; }
    }
}
